{"uuid": "111a2837-dfba-4322-b5b0-0e27de9884a1", "children": ["f997a277-fa5e-4267-b2c4-dc56d553c0c6", "4a37330d-d4b5-4fe3-bffb-33a6d85b537f", "dc28dc34-5b39-4111-87f5-795f6b482e4a"], "befores": [{"name": "authentication", "status": "broken", "statusDetails": {"message": "NameError: name 'time' is not defined\n", "trace": "  File \"C:\\Users\\fahadmuhammad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\fahadmuhammad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\fahadmuhammad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\fahadmuhammad\\Desktop\\Amazon\\conftest.py\", line 20, in authentication\n    login.enter_login_username(test_data.login['guest']['username'])\n  File \"C:\\Users\\fahadmuhammad\\Desktop\\Amazon\\pages\\login_page.py\", line 11, in enter_login_username\n    time.sleep(1)\n"}, "start": 1698169764118, "stop": 1698169764118}], "start": 1698169764118, "stop": 1698169764483}